hack/ci/install-cri-o.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+cd "$(mktemp -d)"
+
```

Comment:
What about doing:

```
dnf module enable cri-o:1.20
dnf install cri-o
```

instead of compiling cri-o?

Reply:
We download the static binary bundle from CRI-O which already contains all runtime dependencies. This is even faster than installing it via `dnf` :)